sdk/identity/azure-identity/src/azure_cli_credential.cpp

```diff
@@ -146,15 +146,15 @@ AccessToken AzureCliCredential::GetToken(
   auto const scopes = TokenCredentialImpl::FormatScopes(tokenRequestContext.Scopes, false, false);
   auto const tenantId
       = TenantIdResolver::Resolve(m_tenantId, tokenRequestContext, m_additionallyAllowedTenants);
+  auto const command = GetAzCommand(scopes, tenantId);
```

Comment:
nit: both variables are used only once, so they now can be inlined - that will also emphasize for the reader that they are not being used in the funcltion anywhere below.
```suggestion
  auto const command = GetAzCommand(
    TokenCredentialImpl::FormatScopes(tokenRequestContext.Scopes, false, false), 
    TenantIdResolver::Resolve(m_tenantId, tokenRequestContext, m_additionallyAllowedTenants));
```

Reply:
That's fair, but I prefer to keep it as names locals for readability. There's already a lot happening at each of the function calls.